.github/workflows/tests.yaml

```diff
@@ -43,3 +43,36 @@ jobs:
         with:
           name: chisel-test-coverage.html
           path: ./*.html
+
+  real-archive-tests:
+    # Do not change to ubuntu-latest as fips-updates may not be available there.
+    runs-on: ubuntu-22.04
+    name: Real Archive Tests
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Run real archive tests
+        env:
+          PRO_TOKEN: ${{ secrets.PRO_TOKEN }}
+        run: |
+          set -ex
+
+          detach() {
+            sudo pro detach --assume-yes || true
+            sudo rm -f /etc/apt/auth.conf.d/90ubuntu-advantage
+          }
+          trap detach EXIT
+
+          # Attach pro token and enable services
+          sudo pro attach ${PRO_TOKEN} --no-auto-enable
+          sudo pro enable fips-updates esm-apps esm-infra --assume-yes
+
+          # Make apt credentials accessible to USER.
+          sudo setfacl -m u:$USER:r /etc/apt/auth.conf.d/90ubuntu-advantage
```

Comment:
It’s because Chisel needs appropriate permissions to read the `/etc/apt/auth.conf.d/90ubuntu-advantage` file. One way to do so would be to run Chisel with root privileges using `sudo`, another way is to do the above – grant the user permission to read the file.

Reply:
What I mean is, if I am a regular user and I want to use Chisel, do I have to run that command as well?

Reply:
Yup, you will need to give chisel read access to the `/etc/apt/auth.conf.d/90ubuntu-advantage` file. You can run chisel with `sudo` or like above, set ACL.

Reply:
TODO: Add some docs in the README about giving the proper permissions to the specific file. Also, add a note about snap.

Reply:
Added docs in README: 94e7fe5.